runtime/astar/src/lib.rs

```diff
@@ -284,7 +286,7 @@ impl pallet_timestamp::Config for Runtime {
 
 parameter_types! {
     pub const BasicDeposit: Balance = deposit(1, 258);  // 258 bytes on-chain
-    pub const FieldDeposit: Balance = deposit(0, 66);  // 66 bytes on-chain
+    pub const ByteDeposit: Balance = deposit(0, 66);  // 66 bytes on-chain
```

Comment:
```suggestion
    pub const ByteDeposit: Balance = deposit(0, 1);
```

Since it's only the amount of deposit per byte, not the entire field.